plugins/c8y_remote_access_plugin/src/config.rs

```diff
@@ -0,0 +1,43 @@
+use miette::Context;
+use miette::IntoDiagnostic;
+use std::net::IpAddr;
+
+pub struct TedgeConfig {
+    pub c8y: TedgeC8yConfig,
+    pub mqtt: TedgeMqttConfig,
+}
+
+pub struct TedgeC8yConfig {
+    pub url: String,
+}
+
+pub struct TedgeMqttConfig {
+    pub port: u16,
+    pub bind_address: IpAddr,
+}
+
+impl TedgeConfig {
+    pub fn read_from_disk() -> miette::Result<Self> {
+        use tedge_config::C8yUrlSetting;
+        use tedge_config::ConfigSettingAccessor;
+        use tedge_config::MqttBindAddressSetting;
+        use tedge_config::MqttPortSetting;
```

Comment:
Quite an unusual style, at least for me. Are there any benefits in keeping these `use` statements at the method level instead of the module?

Reply:
On the contrary, I like this style that avoids to pollute the module namespace.